app/src/main/java/com/infomaniak/mail/data/cache/RealmDatabase.kt

```diff
@@ -229,12 +231,15 @@ object RealmDatabase {
             .migration(MAILBOX_INFO_MIGRATION)
             .build()
 
-        fun mailboxContent(mailboxId: Int, userId: Int) = RealmConfiguration
-            .Builder(mailboxContentSet)
-            .name(mailboxContentDbName(userId, mailboxId))
-            .schemaVersion(MAILBOX_CONTENT_SCHEMA_VERSION)
-            .migration(MAILBOX_CONTENT_MIGRATION)
-            .build()
+        fun mailboxContent(mailboxId: Int, userId: Int): RealmConfiguration {
+            if (mailboxId == DEFAULT_ID) SentryLog.e(TAG, "RealmConfiguration problem with mailbox content, default ID is used.")
```

Comment:
This SentryLog won't show by itself in Sentry.
You should replace it by a `Sentry.captureMessage`